package.json

```diff
@@ -15,7 +15,7 @@
     "install-plugins": "node_modules/7zip-bin-win/x64/7za.exe x plugins/obs-browser-2987-old.7z -onode-obs/",
     "package": "rm -rf dist && build -w --x64 --em.env=production",
     "release": "yarn install --cwd bin && node bin/release.js",
-    "test": "ava",
+    "test": "cd test && tsc && ava",
```

Comment:
We're trying to avoid changing directories in scripts like this.  Can we pass the relative path to `tsc` instead?

Reply:
Actually after the script execution the directory will return to project's root. Will try to pass the relative path

Reply:
K.  It's not a huge deal.